### PR TITLE
INT-149-2: Fix for 'table of contents' disappearing in INT-149 branch.

### PR DESCRIPTION
**Pull request by mari-mcmurtrie**
INT-149 was reverted from dev because table of contents was gone. This is for the fix.

### CODE REVIEW

**Comment by mari-mcmurtrie** [comment]
ArticleWrapperComponent has to have "header" property. @lizlux, will you please CR this? Thank you!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
 ## release-132 (2015-08-12 10:00 UTC)
-* Revert [INT-149](https://wikia-inc.atlassian.net/browse/INT-149) Refactor article contribution buttons into separate component
```
Comment (rogatty) [inline CHANGELOG.md]: Please leave changelog as it is.

**Comment by rogatty** [comment]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (tomasznapieralski) [inline CHANGELOG.md]: +1

Comment (mari-mcmurtrie) [inline CHANGELOG.md]: Aww sorry I did not know. Thank you!

**Pull request merged**